sympy/printing/latex.py

```diff
@@ -34,18 +34,29 @@
 tex_greek_dictionary = {
     'Alpha': 'A',
     'Beta': 'B',
+    'Gamma': r'\Gamma',
+    'Delta': r'\Delta',
     'Epsilon': 'E',
     'Zeta': 'Z',
     'Eta': 'H',
```

Comment:
Well, LaTeX supports symbols like \Eta, despite some symbols being similar in Latin and Greek alphabets, they are not the same. Maybe all symbols should be translated?